deepchem/feat/mol_graphs.py

```diff
@@ -263,6 +263,11 @@ def agglomerate_mols(mols, max_deg=10, min_deg=0):
 
     num_mols = len(mols)
 
+    # Check if the all the objects in the mols array are ConvMol objects
+    for m in mols:
+      if type(m) != ConvMol:
+        raise RuntimeError('Incorrect featurization used for Convolution model')
```

Comment:
I think this should likely be a `ValueError` since it's for an incorrect argument type

Reply:
Sure! I'll change the error type and create a new couple of PRs for the tutorials.